src/tool_writeout.c

```diff
@@ -156,6 +158,11 @@ void ourWriteOut(CURL *curl, struct OutStruct *outs, const char *writeinfo)
                    curl_easy_getinfo(curl, CURLINFO_RESPONSE_CODE, &longinfo))
                   fprintf(stream, "%03ld", longinfo);
                 break;
+              case VAR_HEADER_COUNT:
+                {
+                  fprintf(stream, "%ld", per->header_count);
+                }
```

Comment:
You don't need these braces.

Reply:
>     1. There's no docs for the variable
I will write soon.

>     2. How's it supposed to count amount of headers when one or more redirect are involved?
I think same as `http_code` we should just count the last response header count. I don't know how to do that.

>     3. Test 970 needs update too right?
Yes, after updating the JSON response to include `header_count`. I should update `tool_writeout_json`. As I mentioned before, it currently just works with `curl_easy_getinfo`.